libnavigation-core/api/current.txt

```diff
@@ -463,7 +463,7 @@ package com.mapbox.navigation.core.telemetry.events {
     field public static final String REROUTE = "reroute";
     field public static final String ROAD_CLOSED = "road_closed";
     field public static final String ROAD_IS_MISSING_FROM_MAP = "road_is_missing_from_map";
-    field public static final String ROAD_KNOW_BY_DIFFERENT_NAME = "road_know_by_different_name";
+    field public static final String ROAD_KNOWN_BY_DIFFERENT_NAME = "road_known_by_different_name";
```

Comment:
Will changing this event ID be a problem on the backend? Or should we only rename the category in the UI without touching the underlying event ID?

/cc @baddleydone @shelkovin

Reply:
yep, I thought about it when I made the change. it will depend on whoever uses the metrics. and they can make that decision.